packages/forklift-console-plugin/webpack.config.ts

```diff
@@ -8,7 +8,7 @@ import { TsconfigPathsPlugin } from 'tsconfig-paths-webpack-plugin';
 import { type Configuration as WebpackConfiguration, EnvironmentPlugin } from 'webpack';
 import { type Configuration as WebpackDevServerConfiguration } from 'webpack-dev-server';
 
-import { DynamicConsoleRemotePlugin } from '@kubev2v/webpack';
+import { DynamicConsoleRemotePlugin } from '../webpack/src/DynamicConsoleRemotePlugin';
```

Comment:
This should still be the package name import!

Reply:
oyy, yes, will fix :+1: